### DISCUSSION TITLE
Need help with error message in practical 2-5 run script

### DISCUSSION THREAD

**Discussion by 2199105T** [IBAHCM/RPiR#199] [Archive]
I keep getting this error message saying that my figure margins are too large when running and plotting my script and im not sure how to fix it! Can anyone advise?

Here is the link:
https://github.com/IBAHCM/sarahtonnar_series02/blob/main/0205_run_SIR.R

Thanks,

Sarah

**Comment by richardreeve** [comment]
Does this happen when you are writing the report? Or is it only when you are running the code live in RStudio? If only the latter, it’s probably that your plot pane in RStudio is too small on your screen, and it literally can’t find room to plot everything.

**Reply by 2199105T** [reply]
Yep that seems to be working now, thank you!

**Comment by soniamitchell** [comment]
Also, and unrelatedly, if I click on your main repo page it says "**Sarah Tonnar** and **Sarah Tonnar** Making some formatting changes", which suggests that you might be using two different email addresses. It's not super important, but you might want to check the address you're using in Rstudio matches up with your default on GitHub.

**Reply by 2199105T** [reply]
Hi @soniamitchell, 

Do you  know how  I can check what email address on R?

Sarah x

**Reply by soniamitchell** [reply]
Hi @2199105T, here's a useful link https://happygitwithr.com/hello-git.html

You can check what your config options currently are by running 
```
git config --global --list
```
in the terminal (there's a tab next to the console in Rstudio, which is probably also there if you're using windows?)

You can change your email (if necessary) using

```
usethis::use_git_config(user.name = "Jane Doe", user.email = "jane@example.org")
```
